.github/workflows/smoke-test-daily.yml

```diff
@@ -18,15 +18,17 @@ jobs:
     e2e-tests:
         name: E2E tests on trunk
         runs-on: ubuntu-20.04
-        if: always()
         env:
-            BASE_URL: ${{ secrets.SMOKE_TEST_URL }}
-            ADMIN_USER: ${{ secrets.SMOKE_TEST_ADMIN_USER }}
             ADMIN_PASSWORD: ${{ secrets.SMOKE_TEST_ADMIN_PASSWORD }}
+            ADMIN_USER: ${{ secrets.SMOKE_TEST_ADMIN_USER }}
             ADMIN_USER_EMAIL: ${{ secrets.SMOKE_TEST_ADMIN_USER_EMAIL }}
-            CUSTOMER_USER: ${{ secrets.SMOKE_TEST_CUSTOMER_USER }}
+            ALLURE_REPORT_DIR: ${{ github.workspace }}/plugins/woocommerce/tests/e2e-pw/test-results/allure-report
+            ALLURE_RESULTS_DIR: ${{ github.workspace }}/plugins/woocommerce/tests/e2e-pw/test-results/allure-results
```

Comment:
I added these 2 environment variables so that we'll have control on where to place the Allure reports